test/octalpermissions-success.yml

```diff
@@ -17,3 +17,13 @@
       file:
         path: foo
         mode: 0123
+
+    - name: octal permissions test success (4000)
+      file:
+        path: baz
+        mode: 4000
```

Comment:
@cblakkan Should it be? Maybe I didn't understand the problem fully... Do four digit permissions also need a leading zero?